commit 70dfc29c883588a6bf7d609dd70193f3ce49509d
Author: Omer Bar-or <omer@bar-or.org>
Date:   Mon Feb 15 22:53:16 2010 -0800

    Added check for MagickLibAddendum (replaced MagickLibSubversion).

Applied minus a whitespace-only part that did not apply as-is.

diff --git a/ext/RMagick/extconf.rb b/ext/RMagick/extconf.rb
index 6446d40..78a6260 100644
--- a/ext/RMagick/extconf.rb
+++ b/ext/RMagick/extconf.rb
@@ -233,6 +233,7 @@ have_func("snprintf", headers)
    "LevelColorsImageChannel",        # 6.5.6-4
    "LevelizeImageChannel",           # 6.4.2
    "LiquidRescaleImage",             # 6.3.8-2
+   "MagickLibAddendum",              # 6.5.9-1
    "OpaquePaintImageChannel",        # 6.3.7-10
    "QueueAuthenticPixels",           # 6.4.5-6
    "RemapImage",                     # 6.4.4-0
diff --git a/ext/RMagick/rmagick.h b/ext/RMagick/rmagick.h
index e18a678..8aa6f3f 100644
--- a/ext/RMagick/rmagick.h
+++ b/ext/RMagick/rmagick.h
@@ -162,6 +162,13 @@
 #define DestroyConstitute(void) ConstituteComponentTerminus(void)
 #endif
 
+/** ImageMagick 6.5.9 replaced MagickLibSubversion with
+ * MagickLibAddendum.
+ */
+#if defined(HAVE_MAGICKLIBADDENDUM)
+#define MagickLibSubversion MagickLibAddendum
+#endif
+
 /** IM 6.4.1 replaced AllocateImage with AcquireImage.
  * Both have the same signature.
  */
